regolith/dates.py

```diff
@@ -49,6 +49,14 @@
 }
 
 
+def year_to_int(y):
```

Comment:
I see.  we don't need a ``year_to_int`` function, just do ``int(year)`` in the code.